src/components/layoutComponent/LayoutComponent.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import { Outlet } from 'react-router-dom';
+
+import styles from '@/App.module.css';
+import Footer from '@/components/footer/Footer.component.tsx';
+import Header from '@/components/header/Header.component.tsx';
+
+export default function Main() {
```

Comment:
Name of components should match filename.

Reply:
fix resolve